aesara/configdefaults.py

```diff
@@ -744,6 +744,15 @@ def add_compile_configvars():
         in_c_key=False,
     )
 
+    config.add(
+        "fastmath_opts",
```

Comment:
Added this configuration which will make the rewrite optional.

Reply:
Instead, don't make them canonicalizations (i.e. don't use `register_canonicalize`) and register them manually to a different optimization DB.  As it currently stands, `local_add_sub_collector` is being called unnecessarily far too often.

In general, optimization selection and filtering is accomplished using the optimizations DBs and tags.